app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from datetime import date
+from app.errors import (
+    NotWearingMaskError, NotVaccinatedError, OutdatedVaccineError)

```

Comment:
Please add blank line between build-in and your own imports. Also put bracket to a new line
```suggestion
    NotWearingMaskError, NotVaccinatedError, OutdatedVaccineError
)

```